Cargo.toml

```diff
@@ -20,5 +20,5 @@ log = "0.4.17"
 prost = "0.12"
 prost-types = "0.12"
 protobuf = { version = "3.3" }
-up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", branch = "main" }
+up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", rev = "68c8a1d94f0006daf4ba135c9cbbfddcd793108d" }
```

Comment:
Ah okay, and then if there's a tag, we can use that instead. I'm always super impressed by Rust tooling. Using a git tag has been [supported since 2014](https://github.com/rust-lang/cargo/issues/126#issuecomment-48101852) :slightly_smiling_face: